contracts/src/TokenRemote/ERC20TokenRemote.sol

```diff
@@ -43,10 +64,27 @@ contract ERC20TokenRemote is IERC20TokenTransferrer, ERC20Upgradeable, TokenRemo
         string memory tokenSymbol,
         uint8 tokenDecimals_
     ) public initializer {
+        __ERC20TokenRemote_init(settings, tokenName, tokenSymbol, tokenDecimals_);
+    }
+
+    // solhint-disable-next-line func-name-mixedcase
+    function __ERC20TokenRemote_init(
```

Comment:
Also just for my own understanding: is the only benefit of creating the `__ERC20TokenRemote_init` internal function and calling it from `initialize` to support contracts that may inherit from this contract in the future?

Reply:
yup mostly it, and also to keep a convention for upgradeable contracts